content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -223,7 +223,7 @@ created and deleted with the `kubeadm token` command. See the
 ### Installing a pod network {#pod-network}
 
 You **MUST** install a pod network add-on so that your pods can communicate with
-each other.
+each other. Also it may be necessary to disable firewall on host machine.
```

Comment:
I'm a bit hesitant to suggest that users disable their firewall. Instead we should recommend that they consult the documentation for the networking add-on they are using for any host firewall requirements.